incubator/kube-aws-ingress-controller/templates/daemonset.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: {{ template "kube-aws-ingress-controller.name" . }}-skipper-ingress
+  namespace: kube-system
+  labels:
+    component: ingress
```

Comment:
It doesn't look like you have multiple components. I'd remove component labels.

Make sure all resources have standard labels.